alpha/lib/model/LiveEntry.php

```diff
@@ -629,80 +642,80 @@ public function setMediaServer($index, $hostname, $applicationName = null)
 
 		KalturaLog::debug("about to setMediaServer. index: $index, hostname: $hostname");
 		$this->setLastBroadcast(kApiCache::getTime());
-		$server = new kLiveMediaServer($index, $hostname, $mediaServerNode->getDc(), $mediaServerNode->getId(), $applicationName ? $applicationName : $mediaServerNode->getApplicationName());
-		$this->putInCustomData("server-$index", $server, LiveEntry::CUSTOM_DATA_NAMESPACE_MEDIA_SERVERS);
-		$this->setLiveStatus(LiveEntryStatus::PLAYABLE, $index);
+		$dbLiveEntryServerNode->setStatus(EntryServerNodeStatus::PLAYABLE);
+		$dbLiveEntryServerNode->setServerType($index);
```

Comment:
You already set setServerType & setServerNodeId in the service when creating the object no need to do the same here.